ci/src/cI_web.ml

```diff
@@ -277,8 +280,11 @@ let routes ~logs ~ci ~server ~dashboards =
     ("branch",          fun () -> new branch_list t);
     ("tag",             fun () -> new tag_list t);
     (* Individual targets *)
-    ("pr/:user/:repo/:id",                   fun () -> new pr_page t);
-    ("ref/:user/:repo/:id",                  fun () -> new ref_page t);
+    (":user/:repo/pr/:id",     fun () -> new pr_page t);
+    (":user/:repo/ref/:id",    fun () -> new ref_page None t);
+    (":user/:repo/branch/:id", fun () -> new ref_page (Some "heads") t);
```

Comment:
you mean `ref/$encode(heads/master)` and `branch/master` ? I can filter out the first one, but I think it is more consistent to keep it if client are just interested on raw refs (as some of the refs are neither `branch` or `tags`, e.g. `pull`)